src/D2L.CodeStyle.Analyzers/Immutability/ImmutabilityAnalyzer.cs

```diff
@@ -173,6 +183,54 @@ SimpleNameSyntax syntax
 			}
 		}
 
+		private static void AnalyzeConditionalImmutability(
+			SyntaxNodeAnalysisContext ctx,
+			SyntaxNode syntax
+		) {
+			// Ignore references in DocComments such as crefs
+			if( syntax.IsFromDocComment() ) {
+				return;
+			}
+

```

Comment:
```suggestion
```

I don't think this is necessary, because neither of these kinds of syntax can show up inside a doc comment?